src/limits/TimeLimitListener.cc

```diff
@@ -157,7 +158,7 @@ TimeLimitListener::ProcessTimeUsage TimeLimitListener::getProcessTimeUsage() {
         throw SystemException("Error reading /proc/childPid_/stat");
     }
 
-    ProcessTimeUsage result;
+    ProcessTimeUsage result{};
```

Comment:
j/w, a jeślli już to spacja przed inicjalizatorem

Reply:
to się robi bez spacji zazwyczaj. to jest użycie konstruktora domyślnego, ale nawiasy są inne, żeby różniło się od definicji funkcji.

Reply:
^

Tak samo raczej piszemy
```c++
std::string s("foo");
std::string s{"foo"};
auto s = std::string("foo");
auto s = std::string{"foo"};
```